src/index.ts

```diff
@@ -87,6 +87,7 @@ export class ECRDeployment extends CoreConstruct {
     const handler = new lambda.SingletonFunction(this, 'CustomResourceHandler', {
       uuid: this.renderSingletonUuid(props.memoryLimit),
       code: lambda.Code.fromAsset(path.join(__dirname, '../lambda'), {
+        assetHashType: AssetHashType.SOURCE, // see https://github.com/aws/aws-cdk/pull/12984
```

Comment:
@TRANTANKHOA What will happen if I don't set this? I don't really get it.

Reply:
I find cdk keeps rebuilding the lambda image on repeated builds, which is similar to the issue mentioned. I think change the hashType should fix that

Reply:
btw, I've tried your stack and the go lambda failed for me with error `fork/exec /var/task/main: permission denied: PathError null`. Do you have a hint why that is?

Reply:
Ok, I'll merge this PR firstly. And Please verify if this can solve your problem.  @TRANTANKHOA

Reply:
@TRANTANKHOA Can you create a sample repo how you write your stack? And which region did u deploy?